src/ray/core_worker/core_worker.cc

```diff
@@ -730,6 +730,8 @@ void CoreWorker::Disconnect(
           exit_type, exit_detail, creation_task_exception_pb_bytes));
     }
   }
+  RAY_LOG(INFO) << "Disconnecting to the remote worker.";
+  core_worker_client_pool_->DisconnectAll();
```

Comment:
Also the connection is directional. You have a connection to the remote worker doesn't mean remote worker's pool has a connection to you.

Reply:
This problem should not be affected, because this is an operation of trying to disconnect. If there is no connection in the remote, there will be no problem.